crates/cdk/src/wallet/subscription/ws.rs

```diff
@@ -0,0 +1,147 @@
+use std::collections::HashMap;
+use std::sync::atomic::AtomicUsize;
+use std::sync::Arc;
+
+use futures::{SinkExt, StreamExt};
+use tokio::sync::{mpsc, RwLock};
+use tokio_tungstenite::connect_async;
+use tokio_tungstenite::tungstenite::Message;
+use tracing::{debug, error};
```

Comment:
NIT: Code style we don't import tracing and use the full path when we use it. 

https://github.com/cashubtc/cdk/blob/main/CODE_STYLE.md#full-paths-for-logging